### PR TITLE
show app version in simple About modal

### PR DESCRIPTION
**Pull request by weedySeaDragon**
#### PT URL: https://www.pivotaltracker.com/story/show/136308203

Show the admin the specific version of the system that is running.  Show the info in a simple 'About' box.
For now, only the admin can access it.  We might decide later to provide more info and also show it to visitors/users/members.

## Changes proposed in this pull request:
1.  simple class "AppVersion" gets info from REVISION file (created during capistrano deployment).  fallback = '1.0.0 <revision unknown>'
2.  simple "About" modal is a Bootstrap modal.  This is just a simple version; we can change/refine the design
3.  added it to the Admin's login menu. 

### About modal:
about-v1-0-sv
about-v1-0-en

---
## Ready for review:
@AgileVentures/shf-project-team

### CODE REVIEW

**Comment by weedySeaDragon** [comment]
FYI:  build failed with one of our common intermittently failing cucumber scenarios:
`cucumber features/view_all_companies.feature:206 # Scenario: Pagination`

but the rest passed.

**Comment by patmbolger** [comment]
@weedySeaDragon This looks fine as is.  However, I recommend adding tests to the `admin` menu section of `spec/views/application/_navigation.html.haml_spec.rb`.  We're missing tests for this menu option as well as "Master Checklists", "Dashboard", and "Become this user".

**Review by patmbolger — CHANGES_REQUESTED** [review]
See comment.

File: config/locales/sv.yml [hunk 1]
```diff
@@ -412,6 +412,9 @@ sv:
     progress_bar:
       complete: *complete
 
+  about_info:
+    title: Om detta webbplats
```
Comment (thesuss) [inline config/locales/sv.yml]: Om denna webbplats

**Review by thesuss — APPROVED** [review]
Approved w change

**Pull request merged**